src/flamenco/runtime/fd_runtime_spad.c

```diff
@@ -0,0 +1,26 @@
+#include "fd_runtime_spad.h"
+
```

Comment:
this is nice

Reply:
maybe we add this to util and just generally expose a valloc for spad? There's nothing necessarily runtime specific about this